### PR TITLE
add make test to travis script

### PR DESCRIPTION
**Pull request by labseven**

### CODE REVIEW

**Comment by labseven** [comment]
The travis tests are passing :smile:

**Comment by SeunginLyu** [comment]
Nice! Thanks for integrating my tests

File: src/Makefile [hunk 1]
```diff
@@ -5,11 +5,12 @@ all: shvec.out
...
+	echo "Tests are starting..."
+	./test_shvalloc.out
 
 clean:
 	rm shvec.out
 
-test_shvalloc.out: minunit.h test_shvalloc.c
-	gcc -g -Wall test_shvalloc.c -o test_shvalloc.out
\ No newline at end of file
+test_shvalloc.out: minunit.h test_shvalloc.c shvec.c shvec.h
```
Comment (SeunginLyu) [inline src/Makefile]: wouldn't we have a seperate test for shvec?
I was thinking of _test_shvec.c_
_test_shvalloc_ only includes tests for the custom malloc (shvalloc)

Comment (labseven) [reply src/Makefile]: oh I see. I'll fix this

File: src/Makefile [hunk 2]
```diff
@@ -5,11 +5,20 @@ all: shvec.out
 shvec.out: shvec.c shvec.h
 	gcc $(FLAGS) shvec.c -o shvec.out
 
-test: shvec.c shvec.h
-	echo "Tests are running!"
+test: test_shvalloc test_shvec
```
Comment (SeunginLyu) [inline src/Makefile:8]: oh this is nice

File: src/test_shvec.c [hunk 3]
```diff
@@ -0,0 +1,37 @@
+/*
+To run the test, type in your command line 
+"make test_shvalloc && ./test_shvalloc.out" 
+*/
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <assert.h>
+#include "minunit.h"
+#include <stdint.h>
```
Comment (SeunginLyu) [inline src/test_shvec.c]: we might not need <stdint.h> if we aren't using any uint_64 or uint_32 types

**Review by SeunginLyu — APPROVED** [review]

**Comment by labseven** [comment]
I fixed the two comments, and included shvec.h in test_shvec

**Pull request merged**